### PR TITLE
MC-13916: enable disable cdn waf scripts

### PR DESCRIPTION
**Pull request by anusha-dey**
#### Changes made
-enable/disable CDN, WAF and Scripts

#### Related PRs
- https://github.com/cloudops/cloudmc-stackpath-plugin/pull/151

### CODE REVIEW

File: source/includes/stackpath/_sites.md [hunk 1]
```diff
@@ -204,3 +204,165 @@ Attributes | &nbsp;
...
+curl -X POST \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/api/v1/services/stackpath/test-area/sites/f8ad8351-4f07-4b52-92ec-fd2983873853?operation=enable_cdn"
+```
+> The above command returns a JSON structured like this:
+
+```json
+{
+  "taskId": "57fc8d89-6f13-451b-8b66-fcd96e1fedbd",
+  "taskStatus": "PENDING"
```
Comment (RGitB) [inline source/includes/stackpath/_sites.md]: I seem to be getting success for these operations. Are you getting pending status?

**Review by fyrnaga — APPROVED** [review]

**Review by RGitB — APPROVED** [review]
Left one comment but looks good 👍

File: source/includes/stackpath/_sites.md (hunk 1 above)
Comment (anusha-dey) [reply source/includes/stackpath/_sites.md]: I get success as well, I'll change that

**Review by RGitB — APPROVED** [review]
thanks for the changes

**Pull request merged**